olp-cpp-sdk-core/include/olp/core/client/FetchOptions.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+namespace olp {
```

Comment:
Everything that is under the folder client should be under olp::client.
Nice try, avoiding changing dataservice-read classes using this. Alternative just add this to ApiClientLookup class as nested enum and do a small conversion method. Should be much faster than changing every class in dataservice-read.